src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -276,62 +295,65 @@ export const NavBar: React.FC<NavBarPropsType> = ({ hasBackButton = true }) => {
+                            }
+                            showNormalRadio={showNormalRadio}
+                            showSoundCloudRadio={showSoundCloudRadio}
+                            radioStation={radioStation}
                             isRadioPlaying={isRadioPlaying}
+                            handleRadioEnable={handleRadioEnable}
+                            volume={volume}
+                            setVolume={setVolume}
                           />
                         </Popover.Content>
                       </Popover>
                     }
                     rootClose={true}
                     defaultShow={showRadioOverlay}
                   >
-                    <button
-                      className={`profile-icon navbar-link-radio ${
-                        volume === 0 && "off"
-                      }`}
-                    />
+                    <button className="NavBar__menu--link">
+                      <img src={RadioIcon} alt="radio icon" />
+                    </button>
                   </OverlayTrigger>
                 )}
 
-                {showSoundCloudRadio && (
-                  <S.RadioTrigger>
-                    <button
-                      className={`profile-icon navbar-link-radio ${
-                        volume === 0 && "off"
-                      }`}
-                      onClick={toggleShowRadioPopover}
-                    />
-
-                    <S.RadioWrapper showRadioPopover={showRadioPopover}>
-                      <iframe
-                        title="venueRadio"
-                        id="sound-cloud-player"
-                        scrolling="no"
-                        allow="autoplay"
-                        src={`https://w.soundcloud.com/player/?url=${radioStation}&amp;start_track=0&amp;single_active=true&amp;show_artwork=false`}
-                      />
-                    </S.RadioWrapper>
-                  </S.RadioTrigger>
+                {showRadio && (
+                  <button
+                    className={volumeControlClassname}
+                    onClick={toggleMute}
+                  >
+                    <FontAwesomeIcon icon={volumeIcon} />
+                  </button>
                 )}
                 <div
                   className="navbar-links-user-avatar"
                   onClick={handleAvatarClick}
                 >
-                  <UserAvatar user={userWithId} showStatus size="medium" />
+                  <UserAvatar
+                    user={userWithId}
+                    showStatus
+                    containerClassName="NavBar__userAvatar"
+                  />
                 </div>
+                <OverlayTrigger
+                  trigger="click"
+                  placement="bottom-end"
+                  overlay={MenuPopover}
+                  rootClose={true}
+                >
+                  <button className="NavBar__menu--link">
```

Comment:
Can use `ButtonNG` with `iconOnly` prop set to `true` or other props of it

Reply:
fixed 61d9df7